src/Component/Logo/Logo.js

```diff
@@ -21,14 +21,21 @@ const HorizontalLogo = ({ className = '' }) =>
 // Logo :: Props -> React.Component
 export default ({
   restaurantShown,
+  historiesShown,
 }) =>
   <figure data-is="brand-logo">
     <Link to="/">
-      <HorizontalLogo className="is-hidden-desktop"/>
+      {restaurantShown || historiesShown
+        ? <HorizontalLogo />
+        : null
+      }
 
-      {restaurantShown
-        ? <HorizontalLogo className="is-hidden-touch"/>
-        : <FullWidthLogo/>
+      {!historiesShown && !restaurantShown 
+        ? <>
+          <FullWidthLogo className="is-hidden-touch"/>
+          <HorizontalLogo className="is-hidden-desktop"/>
+        </>
+        : null
```

Comment:
Are you sure the conditions are correct?
There are cases where none of these components aree shown.